### ISSUE TITLE
multi-process support

### ISSUE THREAD

**Issue by adnan-kamili** [cberner/redb#838]
We are planning to use this library, for rust rewrite of our SDK, but for that multi-process support is must. Do you plan to add support for multi-process access anytime soon?

**Comment by cberner** [comment]
No, I don't think it would fit well into redb's design. You'll need to wrap it in an IPC layer

**Comment by cberner** [comment]
Re-opening as I'm now actively working on adding multi-process support for concurrent readers and writers (although only a single writer at a time).

*edit* oh, turns out there was an even old issue. Marking as a duplicate of that
